packages/remark-iframes/src/index.js

```diff
@@ -15,6 +15,9 @@ module.exports = function plugin (opts) {
   function blockTokenizer (eat, value, silent) {
     let eatenValue = ''
     let url = ''
+    if (value[0] !== '!' || value[1] !== '(') {
```

Comment:
`if (value.slice(0, 2) !== '!(')` ?

Reply:
pas faux

Juste pour être sûr : on est certain que value ne peut pas valoir null ou undefined?

Reply:
`value` is always a string. I fixed this and other bugs in #220 .